grafana/overlays/moc/smaug/blackbox-exporter/service-monitor.yaml

```diff
@@ -31,6 +33,8 @@ spec:
           targetLabel: instance
         - replacement: 'https://www.operate-first.cloud/'
           targetLabel: target
+        - replacement: opf-cloud
```

Comment:
could you please change this to `"operate-first.cloud"`? `opf-cloud` seems to vague

Reply:
done